setup-ci.sh

```diff
@@ -15,6 +15,12 @@ read s3regionendpoint
 echo 'Please enter the s3bucketname'
 read s3bucketname
 
+echo 'Please enter the users login name'
+read userusername
```

Comment:
Is there a better way to describe this variable? Like openshift_username or something like that?